stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/user_login/UserLoginFragment.kt

```diff
@@ -41,7 +39,7 @@ class UserLoginFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         setupViews()
         observeStateAndEvents()
-        viewModel.init(getNotificationChannelId())
```

Comment:
No need to parse notification's Intent data here at the login screen. It's done afterwards, after a successful login at HomeFragment.